google_guest_agent/addresses.go

```diff
@@ -143,6 +143,9 @@ func getLocalRoutes(ifname string) ([]string, error) {
 		line = strings.TrimPrefix(line, "local ")
 		line = strings.TrimSpace(line)
 		if line != "" {
+			if !strings.Contains(line, "/") {
+				line = line + "/32"
```

Comment:
Is it possible this will introduce a regression for cases that worked prior to having the mask applied?

Reply:
i can't think of a scenario that would fail, but we are also updating the tests for this, since it wasn't caught.